app/register/patient/patientForm.tsx

```
@@ -28,22 +28,36 @@ const formSchema = z.object({
     .string()
     .min(1, { message: 'Name is required' })
     .max(100, { message: 'Name must be 100 characters or less' }),
-  age: z.preprocess((value) => {
+  age: z.preprocess((value: string) => {
     if (typeof value === 'string') {
       const parsed = parseInt(value, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)